var/spack/repos/builtin/packages/bufr/package.py

```diff
@@ -31,6 +32,9 @@ class Bufr(CMakePackage):
     version("11.5.0", sha256="d154839e29ef1fe82e58cf20232e9f8a4f0610f0e8b6a394b7ca052e58f97f43")
     version("11.4.0", sha256="946482405e675b99e8e0c221d137768f246076f5e9ba92eed6cae47fb68b7a26")
 
+    depends_on("c", type="build")
```

Comment:
Will this work? None of our packages has this yet.

Reply:
removed